assets/app/views/create.html

```diff
@@ -41,13 +41,28 @@
               <h2>Using a Template</h2>
               <p>Templates have predefined resources for quickly creating components. You can customize some options in the next step.</p>
               <h3>Instant Apps</h3>
-              <div class="row">
-                <catalog-template template="template" project="projectName" ng-repeat="template in templatesByTag['instant-app'] | orderBy : 'metadata.name'"></catalog-template>
-                <div class="col-md-4">
-                  <div ng-if="!templatesByTag['instant-app'].length">
-                    <em>There are no instant apps available to create.</em>
-                  </div>
-                  <a class="btn btn-lg btn-primary btn-block" href="project/{{projectName}}/catalog/templates">Browse all templates...</a>
+              <div ng-if="!templatesLoaded && (instantApps | hashSize) === 0">Loading...</div>
+              <div ng-if="templatesLoaded && (instantApps | hashSize) === 0">
+                <em>There are no instant apps available.</em>
+              </div>
+              <div class="catalog">
+                <catalog-template
+                    template="template"
+                    project="projectName"
+                    ng-repeat="template in instantApps | toArray | orderBy : ['metadata.name', 'metadata.namespace']">
+                </catalog-template>
+              </div>
+              <div ng-show="!showAllTemplates && (otherTemplates | hashSize) !== 0" class="show-all-button-container gutter-bottom">
+                <a ng-click="showAllTemplates = true" class="btn btn-lg btn-primary">Show all...</a>
```

Comment:
ah, I think I was confusing this with some other mockups that always showed instant apps and other templates, with each section limited by count, and a "Show all..." button to reveal all the items in that section. If this button is showing other templates, the label text should reflect that. What does the button look like with "Show all templates..." text, without the "btn-lg" class?